tests/integration/targets/module_utils_core/roles/ansibleawsmodule.client/tasks/credentials.yml

```diff
@@ -41,19 +29,6 @@
 ##################################################################################
 # Tests using standard credentials from environment variables
 
-- name: Test basic operation using simple credentials (aws-environment)
```

Comment:
Why remove this test? The env variables are still valid, we should keep this one